### ISSUE TITLE
Calendar Integration

### ISSUE THREAD

**Issue by jaguilar2** [toeverything/AFFiNE#7607]
### Description

I am using Affine for everyday project management and journaling. I would be great to be able to integrate my Google calendar and then take notes that are attached to dates. 

### Use case

I could see many people using this in schools and businesses. I know that academics and students would use this feature.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by lukewhrit** [comment]
I think this is a great feature that I know I would use. However, I think it's crucial this feature is implemented using the iCal/ICS format or something along those lines so calendars besides Google are supported.

**Comment by Qile0317** [comment]
Yes - this is the only thing preventing me from completely switching from notion. Aside from having the notion + notion calendar capabilities, personally I feel like each calendar event can be treated as an entry in an events database. Then other things like a task or project or sprint database can be directly integrated. Would be amazing as well if specifically for google calendar, that tasks can be marked done and be reflected.

**Comment by Idenroad** [comment]
It is a game changer to make an "all-in-one" solution to replace Notion, Taskade and other tools like that.

**Comment by forehalo** [comment]
coming with 0.22

**Comment by Torkalis** [comment]
> coming with 0.22

Hmm, its 0.23.3 and still not there...

**Comment by carlos-otero** [comment]
Not working yet in self hosted? 

<img width="1198" height="602" alt="Image" src="https://github.com/user-attachments/assets/0103f6b3-2c07-4530-bd64-ea6fd7a54f31" />

**Comment by seanmamasde** [comment]
it's not showing up on cloud version as well 

<img width="1479" height="548" alt="Image" src="https://github.com/user-attachments/assets/b3eef788-94ac-4eb7-9a24-c529547398c6" />

**Comment by clemrdh** [comment]
Still not working yet ?

**Comment by gavan-x** [comment]
Not for me.
